perllib/Open311/PopulateServiceList.pm

```diff
@@ -19,6 +22,7 @@ sub process_bodies {
         next unless $body->endpoint;
         next unless lc($body->send_method) eq 'open311';
         $self->_current_body( $body );
+        $self->_current_body_cobrand( $body->get_cobrand_handler );
```

Comment:
Do you need this line any more given the change to default in the class?

Reply:
Nope, good spot!